specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/networkWatcher.json

```diff
@@ -4273,6 +4277,14 @@
             "modelAsString": true
           },
           "description": "Test coverage for the endpoint."
+        },
+        "locationDetails": {
+          "$ref": "#/definitions/ConnectionMonitorEndpointLocationDetails",
+          "description": "Location details is optional and only being used for 'AzureArcNetwork' type endpoints, which can contains availability zone along with region details."
```

Comment:
let's not put avaialbility thing for now. We only put region, for managed monitoring, we would also use Region.

Reply:
updated the description.